### PR TITLE
[Sprint 4] Feature book detail logic #3

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/137461
- Feature book detail logic #3
- Effect : None

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -53,6 +53,7 @@
     <string name="are_you_sure_remove_owner_this_book">Are you sure remove owner this book?</string>
     <string name="are_you_sure_add_owner_this_book">Are you sure add owner this book?</string>
     <string name="do_you_want_to_read_this_book">Do you want to read this book?</string>
+    <string name="do_you_want_cancel_waiting_this_book">Do you want cancel waiting this book?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.do_you_want_cancel_waiting_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 2]
```diff
@@ -43,6 +43,7 @@
     <string name="are_you_sure_remove_owner_this_book">Bạn có chắc chắn xóa chủ sở hữu cuốn sách này?</string>
     <string name="are_you_sure_add_owner_this_book">Bạn có chắc chắn thêm chủ sở hữu cuốn sách này?</string>
     <string name="do_you_want_to_read_this_book">Bạn có muốn đọc cuốn sách này?</string>
+    <string name="do_you_want_cancel_waiting_this_book">Bạn có muốn hủy bỏ chờ đợi cuốn sách này?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.do_you_want_cancel_waiting_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt [hunk 3]
```diff
@@ -54,8 +54,10 @@ class BookDetailActivity : BaseActivity(), BookDetailContract.ViewModel, ItemOwn
   private var mUserId: Int? = null
   private var mBookId: Int? = null
   private var mOwners = arrayListOf<Owner>()
-  private var mReadingBook = ReadingBook()
-  private var mStatusWaitingBook: Int = 1
+  private var mUsersWaiting = arrayListOf<User>()
+  private var mActionBookDetail = ActionBookDetail()
+  private var mStatusWantToReadThisBook: Int = 1
+  private var mStatusCancelWaitingThisBook: Int = 5
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt]: pls define enum for status

File: app/src/main/java/com/framgia/fbook/data/source/remote/api/error/BaseException.kt [hunk 4]
```diff
@@ -10,7 +10,8 @@ import java.net.UnknownHostException
  * Created by le.quang.dao on 10/03/2017.
  */
 
-class BaseException : RuntimeException {
+class
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/fbook/data/source/remote/api/error/BaseException.kt]: pls keep old file

File: app/src/main/java/com/framgia/fbook/data/model/ActionBookDetail.kt [hunk 5]
```diff
@@ -18,3 +18,4 @@ class ReadingBook : BaseRequest() {
   @SerializedName("status")
   var status: Int? = null
 }
+
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/fbook/data/model/ActionBookDetail.kt]: remove blank line

**Pull request merged**